queue_management/models/queue_token_location.py

```diff
@@ -46,6 +51,46 @@ class QueueTokenLocation(models.Model):
         "queue.token.location.action", inverse_name="token_location_id"
     )
 
+    @api.model
+    def autocancel(self, hours=24):
```

Comment:
use **kwargs and pass it to timedelta this way we can reuse the same functionality :smile: